test/jit/test_torchbind.py

```diff
@@ -349,3 +349,26 @@ def test_lambda_as_constructor(self):
 
         obj_swap = torch.classes._TorchScriptTesting._LambdaInit(4, 3, True)
         self.assertEqual(obj_swap.diff(), -1)
+
+    def test_default_args(self):
+        def fn() -> int:
+            obj = torch.classes._TorchScriptTesting._DefaultArgs()
+            obj.increment(5)
+            obj.decrement()
+            obj.decrement(2)
+            obj.scale_add(5)
+            obj.scale_add(3, 2)
+            return obj.increment()
+
+        self.checkScript(fn, ())
+
+        def gn() -> int:
```

Comment:
should we also test this case outside of torchscript?

Reply:
`checkScript` tests that, right?

Reply:
oh right, my bad